duxbay.conf

```diff
@@ -32,6 +32,11 @@ LIPATH=${LUSER}/ingest
 
 SPK_EXEC='400'
 SPK_EXEC_MEM='2048m'
+SPK_DRIVER_MEM=''
```

Comment:
They should update with new values. That's going to depend on cluster capacity and YARN configuration.